letsencrypt/rootfs/etc/cont-init.d/file-structure.sh

```diff
@@ -82,3 +82,16 @@ fi
 if bashio::config.exists 'dns.cloudxns_secret_key'; then
       bashio::addon.option 'dns.cloudxns_secret_key'
 fi
+
+# ClouDNS
+# Only a single non-empty auth option must be in /data/dnsapikey when using ClouDNS to avoid a certbot error
+# Code below removes unused auth types from /data/dnsapikey
+if ! bashio::config.has_value 'dns.cloudns_auth_id'; then
+      sed -i '/.*dns_cloudns_auth_id.*/d' /data/dnsapikey
```

Comment:
Instead of deleting, how about removing it from the echo above and only set it if the config exists/has_value? Essentially similar to `dns.google_domains_zone` above.